File: 07.10-Project/FrontendApp/src/app/models/medicament.model.ts
import { MedicamentOrder } from "./medicament-order.model"

export interface Medicament {

  id: number;
  name: string;
  medicamentQty: number;

 // medicamentOrders: MedicamentOrder[];
 // public Medicament() { medicamentOrders }
  //      {  MedicamentOrders = new List<MedicamentOrderModel>();   }
  //public List < MedicamentOrderModel > MedicamentOrders { get; set; }
}

File: 07.10-Project/BackendApp/Models/MedicamentModel.cs
using System.Collections.Generic;
 
namespace BackendApp.Models
{
    public class Medicament
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int MedicamentQty { get; set; }

        public Medicament()
        {
            MedicamentOrders = new List<MedicamentOrderModel>();
        }
        public List<MedicamentOrderModel> MedicamentOrders { get; set; }
}
}
